java/device/iothub-java-client/src/main/java/com/microsoft/azure/iothub/transport/mqtt/MqttIotHubConnection.java

```diff
@@ -43,6 +44,7 @@
     private static final int mqttVersion = 4;
     private static final boolean setCleanSession = false;
     private static final int qos = 1;
+    private static final String[] enabledProtocols = new String[]{"TLSv1"};
 
```

Comment:
Can there be multiple protocols enabled at the same time ? If not can you change this from string array to string ? If so, please leave it as is.